e2e/image/pull_test.go

```diff
@@ -0,0 +1,75 @@
+package image
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/gotestyourself/gotestyourself/icmd"
+	"github.com/stretchr/testify/require"
+)
+
+const notaryURL = "https://notary-server:4443"
+const registryPrefix = "registry:5000"
+
+const alpineImage = "registry:5000/alpine:3.6"
+const busyboxImage = "registry:5000/busybox:1.27.2"
+
+func TestPullWithContentTrust(t *testing.T) {
+	image := createTrustedRemoteImage(t, "trust", "latest")
+
+	// test that pulling without the tag defaults to latest
+	imageWithoutTag := strings.TrimSuffix(image, ":latest")
+	icmd.RunCmd(trustedCmdNoPassphrases(icmd.Command("docker", "pull", imageWithoutTag))).Assert(t, icmd.Success)
+	icmd.RunCommand("docker", "rmi", image).Assert(t, icmd.Success)
```

Comment:
We can test "pulling without the tag defaults to latest" in a unit test, doesn't need to be a case in the e2e suite.

Reply:
if you'd like we can remove it, though I think there's still value in testing this in the trust suite to ensure that there isn't any further manipulation on the default tag.

There's an existing unit tests for this in the image pull tests, output here: https://github.com/docker/cli/blob/master/cli/command/image/testdata/pull-command-success.simple-no-tag.golden